### ISSUE TITLE
Train new nucleus model for max projection

### ISSUE THREAD

**Issue by ninanikitina** [mal-boisestate/afilament#49]

**Comment by ninanikitina** [comment]
Model training information:
        Epochs:          200
        Batch size:      1
        Learning rate:   0.001
        Training size:   26
        Validation size: 4
        Checkpoints:     True
        Device:          cuda
        Images scaling:  1.0
        LOGS: Apr17_17-33-05_LAPTOP-ISSFJ09KLR_0.001_BS_1_SCALE_1.0



![Image](https://user-images.githubusercontent.com/68862634/232818826-0397b57d-90d0-4a89-9db0-a0dfeed4e474.png)



![Image](https://user-images.githubusercontent.com/68862634/232818859-451098e2-cf3b-4ec6-a521-f805fb9301ed.png)

**Comment by ninanikitina** [comment]
Add images that did not go through the test to the data set:

   - [x] Zeiss confocal 40x control - 1 img
   - [x] Zeiss confocal 40x LIV- 3 img

**Comment by ninanikitina** [comment]
Despite increasing the size of the training set and including the problematic images from MODEL 1, the new model (MODEL 2) still failed to accurately recognize the same images. For example, the image depicted below was included in the training set for MODEL 2, but the new model still failed to recognize it. I suspect that the issue may be related to the image being randomly selected for validation during training. AI plan to train a new model (MODEL 3) without using a validation set during training in order to address this issue.



![Image](https://user-images.githubusercontent.com/68862634/233462415-931434dc-322c-44bb-9be3-45cea1a718a8.png)

**Comment by ninanikitina** [comment]
MODEL 3
INFO: Starting training:
        Epochs:          200
        Batch size:      1
        Learning rate:   0.001
        Training size:   33
        Validation size: 0
        Checkpoints:     True
        Device:          cuda
        Images scaling:  1.0

![Image](https://user-images.githubusercontent.com/68862634/233712392-456c8b56-e332-4580-9a2b-031e0f81158a.png)


![Image](https://user-images.githubusercontent.com/68862634/233712780-753050f6-06c8-4775-a8fd-e4c6e0893b5b.png)
